### DISCUSSION TITLE
pytest: force_terminal by global variable?

### DISCUSSION THREAD

**Discussion by nschloe** [Textualize/rich#2730] [Q&A] [answered]
In https://github.com/Textualize/rich/discussions/1298, it is answered how the color/style codes can be preserved under pytest output. Anwer: Initialize the `Console` like

```python
from rich.console import Console

console = Console(force_terminal=True)
```

I'm wondering if there's a way to achieve this without changing production code, e.g., setting an environment variable, or some other global thing that I could set?

**Comment by willmcgugan** [comment, marked as answer]
`FORCE_COLOR=1` should do it. See [here](https://rich.readthedocs.io/en/latest/console.html#environment-variables) for other env vars.
